src/storage/MainCustomArrayList.java

```diff
@@ -0,0 +1,30 @@
+package storage;
+
+public class MainCustomArrayList {
+    public static void main(String[] args) {
```

Comment:
`main(...)` method into the application can be just 1

Reply:
I decided to delete both of the "Main..." files to resolve this problem because I used them to check how CustomHashSet and CustomArrayList work, but I will be able to replace them by unittests only after switching the project to Maven.

Reply:
Add example of how to use your collections just in your main method in TicketService, at this moment it will be enough)